.travis.yml

```diff
@@ -12,7 +12,7 @@ services:
  - docker
 before_install:
  - sudo apt-get -qq update
- - sudo apt-get install -y libsystemd-dev
+ - sudo apt-get install -y libsystemd-dev gcc-aarch64-linux-gnu
```

Comment:
Can you add this to the README.md as well. There are instructions to install libsystemd-dev there.